src/net_processing.h

```diff
@@ -217,6 +217,9 @@ class PeerManager final : public CValidationInterface, public NetEventsInterface
     /** Send a version message to a peer */
     void PushNodeVersion(CNode& pnode, int64_t nTime);
 
+    /** Send a ping message on a regular schedule or if requested manually */
+    void MaybeSendPing(CNode& pto);
```

Comment:
I don't think exposing more implementation details in net_processing.h is a good idea. Fortunately it's fixable: #20758

Reply:
Totally agree! I think #20758 can happen before or after this PR.

Reply:
Yes, I don't think either need gate the other, the above was literally my thought process in getting to that PR -- sipa may not have nerd sniped me with twitter maths, but that doesn't mean I didn't get nerd sniped anyway...

Reply:
This seems like a more productive form of aj-sniping.

Reply:
I like this kind of aj-sniping.